modular_zubbers/code/game/objects/items/holy_weapons.dm

```diff
@@ -35,3 +35,145 @@
 /obj/item/storage/box/holy/papal/PopulateContents()
 	new /obj/item/clothing/head/mitre(src)
 	new /obj/item/clothing/suit/chaplainsuit/armor/papal(src)
+
+///////////////////////////////
+//// HYPEREUTACTIC BLADE /////
+/////////////////////////////
+/obj/item/dualsaber/chaplain
+	icon = 'modular_zubbers/icons/obj/weapons/1x2.dmi'
+	icon_state = "hypereutactic"
+	worn_icon = 'modular_zubbers/icons/mob/clothing/belt.dmi'
+	inhand_icon_state = "hypereutactic"
+	lefthand_file = 'modular_zubbers/icons/mob/inhands/64x64_lefthand.dmi'
+	righthand_file = 'modular_zubbers/icons/mob/inhands/64x64_righthand.dmi'
```

Comment:
```suggestion
	// Original from Citadel by Sishen1542
	icon = 'modular_zubbers/icons/obj/weapons/1x2.dmi'
	icon_state = "hypereutactic"
	worn_icon = 'modular_zubbers/icons/mob/clothing/belt.dmi'
	inhand_icon_state = "hypereutactic"
	lefthand_file = 'modular_zubbers/icons/mob/inhands/64x64_lefthand.dmi'
	righthand_file = 'modular_zubbers/icons/mob/inhands/64x64_righthand.dmi'
```

Reply:
Oh true, reminds me I should probably make a .txt credits file once I've compiled all the various clockwork cult changes I'm planning for after the freeze is lifted.